python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -1173,7 +1205,17 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
                 for outputRefName in outputRefs.keys():
                     if outputRefName.endswith("Plot") and f"Cycle{cycle}" in outputRefName:
                         ref = getattr(outputRefs, outputRefName)
-                        plotHandleDict[outputRefName] = ref
+                        if isinstance(ref, (tuple, list)):
+                            if "physical_filter" in ref[0].dimensions:
+                                for filterRef in ref:
+                                    handleDictKey = outputRefName + f"_{filterRef.dataId['physical_filter']}"
```

Comment:
```suggestion
                                    handleDictKey = f"{outputRefName}_{filterRef.dataId['physical_filter']}"
```

Reply:
Also, I hope nothing down in `fgcm` tries to split these on `_` and interpret the terms by index, because we don't ban those from `physical_filter` and `band` names.

Reply:
Nothing tries to split on these at all, it's all deterministic.  Before I had to protect the filter names from special characters because they were being used as dataset types, but now the dataset types are clean.